src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -103,51 +104,78 @@ export const MyProfile = () => {
+                                  size="small"
+                                  variant="filled"
+                                />
+                              )}
+                            </Field>
+                            <Field name={'preferredLastName'}>
+                              {({ field }: FieldProps<string>) => (
+                                <TextField
+                                  {...field}
+                                  data-testid={dataTestId.myPage.myProfile.preferredLastNameField}
+                                  id={field.name}
+                                  disabled={!editPreferredNames || isSubmitting}
+                                  label={t('my_page.my_profile.preferred_last_name')}
+                                  size="small"
+                                  variant="filled"
+                                />
+                              )}
+                            </Field>
+                            <Tooltip title={t('common.edit')}>
+                              <IconButton
+                                data-testid={dataTestId.myPage.myProfile.editPreferredNameButton}
+                                onClick={() => setEditPreferredNames(!editPreferredNames)}>
+                                <EditIcon sx={{ width: '1.2rem' }} />
+                              </IconButton>
+                            </Tooltip>
+                          </Box>
+                          <TextField
+                            sx={{ width: 'fit-content' }}
+                            value={personTelephone}
+                            data-testid={dataTestId.myPage.myProfile.telephoneField}
+                            disabled
+                            label={t('my_page.my_profile.telephone')}
+                            size="small"
+                            variant="filled"
+                          />
                         </Box>
-                        <TextField
-                          sx={{ width: 'fit-content' }}
-                          value={personTelephone}
-                          data-testid={dataTestId.myPage.myProfile.telephoneField}
-                          disabled
-                          label={t('my_page.my_profile.telephone')}
-                          size="small"
-                          variant="filled"
-                        />
+
+                        <>
+                          <Box
+                            sx={{
+                              gridArea: 'profile-picture',
+                            }}>
+                            <Typography variant="h3" sx={{ alignSelf: 'start', my: '1rem' }}>
+                              {t('my_page.my_profile.profile_picture')}
+                            </Typography>
+                            <ProfilePictureUploader id={personId} />
+                          </Box>
+                        </>
```

Comment:
husk å ta en egen runde på egen PR også, lurer seg ofte inn sånne ting som man først ser i etterkant :)
```suggestion
                          <Box
                            sx={{
                              gridArea: 'profile-picture',
                            }}>
                            <Typography variant="h3" sx={{ alignSelf: 'start', my: '1rem' }}>
                              {t('my_page.my_profile.profile_picture')}
                            </Typography>
                            <ProfilePictureUploader id={personId} />
                          </Box>
```